permits/forms.py

```diff
@@ -958,9 +962,11 @@ def __init__(self, *args, **kwargs):
             self.fields["status"].choices = tuple(filter2)
 
             if not config.ENABLE_GEOCALENDAR:
-                self.fields["is_public"].widget = forms.HiddenInput()
                 self.fields["shortname"].widget = forms.HiddenInput()
 
+            if not services.is_publication_enabled_for_wots(self.instance):
```

Comment:
Should be in `if not config.ENABLE_GEOCALENDAR` bloc?

Reply:
No, the methods `has_publication_enabled_for_wots` now include check for geocalendar activation